plugins/manager/lib/yform/manager/field.php

```diff
@@ -154,7 +154,13 @@ public function isHiddenInList()
     // sobald die yform value klassen umgebaut worden sind.
     public function toArray()
     {
-        return $this->values;
+        $keys = array_unique(
+            array_merge(
+                ['id','table_name','prio','type_id','type_name','db_type','list_hidden','search','name','label','not_required'],
```

Comment:
Danke für den PR .. Ich fände es besser wenn diese Felder als Konstante in die field Klassen kommen.

Reply:
Hallo Jan, aus irgend einem Grunde habe ich die Branch dieses PR bei mir im Fork nicht mehr verfügbar. Möglicherweise habe ich den Fork mit einem neueren wegen anderer PRs überschrieben. Ich kriege die Anpassung also nicht wie gewünscht rein.

Reply:
kein Thema.. ich übernehme das so .. ist ja nicht viel